test/ejabberd_tests/tests/connect_SUITE.erl

```diff
@@ -511,11 +519,16 @@ mk_value_for_tls_config_pattern() ->
     {tls_config, "{certfile, \"" ++ ?CERT_FILE ++ "\"}, tls,"}.
 
 mk_value_for_compression_config_pattern() ->
-    {tls_config, "{certfile, \"" ++ ?CERT_FILE ++ "\"}, starttls_required,  {zlib, 10000},"}.
+    {tls_config, "{certfile, \"" ++ ?CERT_FILE ++ "\"}, " ++
+                 "starttls_required,  {zlib, 10000},"}.
 
 mk_value_for_starttls_required_config_pattern() ->
     {tls_config, "{certfile, \"" ++ ?CERT_FILE ++ "\"}, starttls_required,"}.
 
+mk_value_for_starttls_required_dh_file_config_pattern() ->
+    {tls_config, "{certfile, \"" ++ ?CERT_FILE ++ "\"}, " ++
+                 "starttls_required, {dhfile, \"" ++ ?DH_FILE ++ "\"},"}.
```

Comment:
I'd move the dhfile param to the function above. Also it'd good to have this param available in default MongooseIM config.